metricbeat/mb/testing/testdata.go

```diff
@@ -447,7 +452,11 @@ func server(t *testing.T, path string, url string) *httptest.Server {
 		}
 
 		if r.URL.Path+query == url {
-			w.Header().Set("Content-Type", "application/json;")
+			if contentType != "" {
+				w.Header().Set("Content-Type", contentType)
+			} else {
+				w.Header().Set("Content-Type", "application/json;")
```

Comment:
Nit. Is trailing `;` needed?
```suggestion
				w.Header().Set("Content-Type", "application/json")
```

Reply:
removed trailing ;